hw/dv/tools/dvsim/common_sim_cfg.hjson

```diff
@@ -29,7 +29,8 @@
                         "^Assert failed: ",
                         "^\\s*Offending '.*'",
                         "^TEST FAILED (UVM_)?CHECKS$",
-                        "^Error:.*$"]  // ISS errors
+                        "^ERROR:.*$",          // ISS errors from CC files
+                        "^[a-zA-Z]*Error:.*$"] // ISS errors from py files
```

Comment:
I think these are generic enough error patterns that may also be adopted by other scripts that may be running along with our simulations. As such, I don't think we need to explicitly say that these are specific to some ISS.